### PR TITLE
DBZ-3027 - Remove POC references from UI

### PR DESCRIPTION
**Pull request by mdrillin**
Initial changes for removing POC references.  The repo name will need to be changed to 'debezium-ui'.  Also the dockerhub image 'debezium-ui-poc'

### CODE REVIEW

File: backend/pom.xml [hunk 1]
```diff
@@ -3,12 +3,12 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>io.debezium.uipoc</groupId>
-    <artifactId>debezium-uipoc-parent</artifactId>
+    <groupId>io.debezium.ui</groupId>
```
Comment (rk3rn3r) [inline backend/pom.xml:6]: @gunnarmorling should we align that groupId to the package namespace of the Java application?

**Review by rk3rn3r — APPROVED** [review]
LGTM

**Comment by mdrillin** [comment]
@rk3rn3r does your question to @gunnarmorling need an answer, or should we merge this?  I was going to merge then rename the repo - but looks like the rename has already been done.

File: backend/pom.xml (hunk 1 above)
Comment (gunnarmorling) [reply backend/pom.xml:6]: Or maybe even io.debezium, like everything else? I don't think it needs a separate group id actually.

Comment (rk3rn3r) [reply backend/pom.xml:6]: Makes sense. Let's do that @mdrillin

**Comment by rk3rn3r** [comment]
@mdrillin can you make the above change to the maven group id, then we are good to go

**Review by gunnarmorling — APPROVED** [review]

**Comment by gunnarmorling** [comment]
Thx, @mdrillin!

**Pull request merged**